core/src/main/protobuf/MLFlowServeRequest.proto

```diff
@@ -33,24 +33,29 @@ message MLFlowServeRequest {
       */
     required string path = 1;
 
+    /**
+     * Relative artifact path. Relative to `path`
+     */
+    optional string artifact_path = 2;
+
     /**
       * path may use a protocol that supports implicit FQRV extraction otherwise fqrv.proto is required
       */
-    optional FQRV fqrv = 2;
```

Comment:
Protocol buffer numbering **must** be maintained for backwards/forwards compatibility. If you must make changes here, and I would argue that's also debatable given this file should match the MLFlow model specification so it's not up to us what goes in here, but if you must, please look into the Protocol buffer backwards/forwards compatibility concerns (including what you need to do for default values). Additionally, if there is a situation where compatibility is not possible then a means to "evolve" schema must be introduced for AKKA serialization and persistence. See Schema Evolution in AKKA: https://doc.akka.io/docs/akka/current/persistence-schema-evolution.html